### PR TITLE
[RFC] Add issue/PR templates

### LINKED ISSUE #5281: Add COC, contributing, issue and PR templates
Should we add a code of conduct to our project? Most projects seem to adopt the [Contributor Covenant](https://www.contributor-covenant.org/) these days. Thoughts? If we decide to adopt this as well, how can people report violations? I assume opening an issue is not the best way to do this, but I don't know if we have a project email address.

### PR DESCRIPTION
**Pull request by massich**
fix #5281 

thake a look at [pandas' templates](https://github.com/pandas-dev/pandas/tree/master/.github)

### CODE REVIEW

**Comment by cbrnr** [comment]
I suggest we add two issue templates, one for bug reports and one for feature requests. We can modify the default templates GitHub already provides to ask for a code snippet, the output of `mne.sys_info()`, and optionally a test file to reproduce the problem.

For the PR template, I'd use the minimal default template [here](https://help.github.com/articles/about-issue-and-pull-request-templates/).

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+<!--
+If your issue is a usage question, submit it here instead:
+- The imbalanced learn gitter: https://gitter.im/scikit-learn-contrib/imbalanced-learn
```
Comment (agramfort) [inline .github/ISSUE_TEMPLATE.md]: change channel

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
+<!--
+If your issue is a usage question, submit it here instead:
+- The imbalanced learn gitter: https://gitter.im/mne-tools/mne-python
+-->
+
+<!-- Instructions For Filing a Bug: https://martinos.org/mne/stable/faq.html#i-think-i-found-a-bug-what-do-i-do -->
```
Comment (massich) [inline .github/ISSUE_TEMPLATE.md]: Shall we improve this part of the FAQ?

File: .github/ISSUE_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+#### Description
+<!-- Example: Joblib Error thrown when calling fit on LatentDirichletAllocation with evaluate_every > 0-->
```
Comment (massich) [inline .github/ISSUE_TEMPLATE.md]: Can someone provide an example that resonates with MNE community?

File: .github/ISSUE_TEMPLATE.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+
+#### Steps/Code to Reproduce
+<!--
+Example:
```
Comment (massich) [inline .github/ISSUE_TEMPLATE.md]: idem, could I get an MWE related that reflects the description.

Maybe a link or short description of MWE should be provided there. WDYT?

**Comment by massich** [comment]
closed in favor of #5413

**Pull request closed without merging**